dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/CompanyOwnershipChecker.kt

```diff
@@ -56,4 +58,44 @@ class CompanyOwnershipChecker(
         val companyId = dataMetaInformationManager.getDataMetaInformationByDataId(dataId).company.companyId
         return isCurrentUserCompanyOwnerForCompany(companyId)
     }
+
+    /**
+     * Method to check whether a given company has at least one company owner
+     * @param companyId the ID of the company
+     * @return a Boolean indicating whether the company has at least one company owner
+     */
+    @Transactional(readOnly = true)
```

Comment:
Kannst du glaube ich entfernen, weil hier keine DB Operation stattfindet.

Reply:
removed

Reply:
Es gibt noch zwei "@Transactional" tags hier im File.  ICh denke die können auch weg, weil darunter ohnehin keine DB Operationen passieren.

Reply:
removed "@transactional" from the two olde functions